### PR TITLE
Unmutes testlogger if test verbose output is enabled

### PR DESCRIPTION
**Pull request by AlexanderYastrebov**
Signed-off-by: Alexander Yastrebov <alexander.yastrebov@zalando.de>

### CODE REVIEW

File: logging/loggingtest/logger.go [hunk 1]
```diff
@@ -96,8 +97,7 @@ func New() *Logger {
 	mute := make(chan bool)
 	quit := make(chan struct{})
 
-	// start muted to reduce spam during failing tests
-	muted := true
+	muted := !testing.Verbose()
```
Comment (AlexanderYastrebov) [inline logging/loggingtest/logger.go:100]: Within #1825 I've observed the problem that was not apparent unless testlogger is unmuted.

File: logging/loggingtest/logger.go [hunk 2]
```diff
@@ -96,8 +97,7 @@ func New() *Logger {
 	mute := make(chan bool)
 	quit := make(chan struct{})
 
-	// start muted to reduce spam during failing tests
```
Comment (AlexanderYastrebov) [inline logging/loggingtest/logger.go:99]: I think rich log output is exactly what is needed to debug the failing test. This PR proposes to unmute when verbose test output is enabled which requires re-running failing test with `go test -v`. Maybe we should simply unmute testlogger unconditionally.

**Comment by szuecs** [comment]
:+1:

**Comment by AlexanderYastrebov** [comment]
:+1:

**Pull request merged**